### PR TITLE
[MU4] Fixed install for Windows and Qt 5.15

### PR DESCRIPTION
**Pull request by igorkorsukov**
For Qt 5.15 need deploy Qt5QmlWorkerScript

### CODE REVIEW

**Review by vpereverzev — APPROVED** [review]

File: main/main.cpp [hunk 1]
```diff
@@ -31,8 +31,12 @@
 #include <windows.h>
 #endif
 
+#include <iostream>
+
 int main(int argc, char** argv)
 {
+    std::cout << "Hello world, I am MuseScore!!\n";
```
Comment (Jojo-Schmitz) [inline main/main.cpp:38]: Serious? This is what `qDebug()` is for

File: main/main.cpp [hunk 2]
```diff
@@ -84,6 +88,7 @@ int main(int argc, char** argv)
 #ifdef BUILD_UI_MU4
     mu::appshell::AppShell app;
     int code = app.run(argcFinal, argvFinal, moduleSetup);
+    std::cout << "after mu::appshell::AppShell app.run, code: " << code << "\n";
```
Comment (Jojo-Schmitz) [inline main/main.cpp:91]: same here

File: mu4/appshell/appshell.cpp [hunk 3]
```diff
@@ -37,6 +38,7 @@ AppShell::AppShell()
 
 int AppShell::run(int argc, char** argv, std::function<void()> moduleSetup)
 {
+    std::cout << "begin AppShell::run\n";
```
Comment (Jojo-Schmitz) [inline mu4/appshell/appshell.cpp:41]: and here

File: mu4/appshell/appshell.cpp [hunk 4]
```diff
@@ -72,10 +74,17 @@ int AppShell::run(int argc, char** argv, std::function<void()> moduleSetup)
     QObject::connect(engine, &QQmlApplicationEngine::objectCreated,
                      &app, [url](QObject* obj, const QUrl& objUrl) {
         if (!obj && url == objUrl) {
+            std::cout << "failed Qml load\n";
```
Comment (Jojo-Schmitz) [inline mu4/appshell/appshell.cpp:77]: and here

File: mu4/appshell/appshell.cpp [hunk 5]
```diff
@@ -72,10 +74,17 @@ int AppShell::run(int argc, char** argv, std::function<void()> moduleSetup)
...
             QCoreApplication::exit(-1);
         }
     }, Qt::QueuedConnection);
 
+    QObject::connect(engine, &QQmlEngine::warnings, [](const QList<QQmlError> &warnings) {
+        for (const QQmlError& e : warnings) {
+            std::cout << "error: " << e.toString().toStdString() << "\n";
```
Comment (Jojo-Schmitz) [inline mu4/appshell/appshell.cpp:84]: and here

File: main/main.cpp (hunk 1 above)
Comment (igorkorsukov) [reply main/main.cpp:38]: Was a problem, Windows build doesn't start, without any messages. qDebug() doesn't print messages to console. 
I will check this again and will fix it if not so.

**Pull request merged**